crates/deploy/Cargo.toml

```diff
@@ -22,3 +22,4 @@ walkdir = { workspace = true }
 prost = "0.11"
```

Comment:
Let's put all dependencies in the root Cargo.toml (and use {workspace=true}), otherwise we'll never be able to de-dupe them